src/php/Command/ReplCommand.php

```diff
@@ -55,24 +55,31 @@ public function run(): void
     private function loopReadLineAndAnalyze(): void
     {
         while (true) {
-            $this->io->output("\e[?2004h"); // Enable bracketed paste
-            $input = $this->io->readline('>>> ');
-            $this->io->output("\e[?2004l"); // Disable bracketed paste
-            $this->checkInputAndAnalyze($input);
+            try {
+                $this->io->output(self::ENABLE_BRACKETED_PASTE);
+                $input = $this->io->readline(self::PROMPT);
+                $this->io->output(self::DISABLE_BRACKETED_PASTE);
+
+                try {
+                    $this->checkInputAndAnalyze($input);
+                } catch (ExitException $e) {
+                    break;
+                }
+            } catch (Throwable $e) {
+                $this->io->output($e->getMessage() . PHP_EOL);
```

Comment:
Is the message enough or would it be better to show the complete stack trace?

Reply:
Full stack trace would be cool for debugging purposes... but a message is also fine... idk

Reply:
Yeah, you're right. `getTraceAsString` is nicer than `getMessage`, I will change it :)